### ISSUE TITLE
Reusable block: Add "Convert to regular blocks" in the ellipsis drop down.

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/gutenberg#32217]
## What problem does this address?

In WordPress 5.6.2. We see text clearly in the toolbar how to convert a Reusable block back to a regular block.

<img width="668" alt="Screen Shot 2021-05-25 at 20 01 04" src="https://user-images.githubusercontent.com/5323259/119546209-f6990e00-bd93-11eb-873c-c0feec5da869.png">

In WordPress 5.7.2 we see the following toolbar for Reusable blocks. The icon does not bring the same clarity as text from the earlier version. It is easy to overlook the icon.

<img width="665" alt="Screen Shot 2021-05-25 at 19 57 08" src="https://user-images.githubusercontent.com/5323259/119545784-7c688980-bd93-11eb-97dd-e30d3f6f8cb5.png">

## What is your proposed solution?

Comparing with converting to a Group block.
Converting a block to a Group can be done two ways through the current block toolbar.
Through the "Transform to" panel block (icon) 

![Screen Shot 2021-05-25 at 20 19 54](https://user-images.githubusercontent.com/5323259/119548750-d159cf00-bd96-11eb-9606-259b806b857f.png)

and in the ellipsis drop down (text).

![Screen Shot 2021-05-25 at 20 20 02](https://user-images.githubusercontent.com/5323259/119549130-36adc000-bd97-11eb-9ffe-a8828513ab51.jpg)

The icon in WP 5.7 is not so obvious and does not express clearly what it does.
It would be helpful to add "Convert to regular blocks" also to the ellipsis drop down.
So the users have the option to click the icon or click the text inside the ellipsis drop down.

![Add-Convert-to-regular-blocks-in-ellipsis-drop-down](https://user-images.githubusercontent.com/5323259/119547890-f26df000-bd95-11eb-913d-7dc40b89e0d8.jpg)

The reason why I am following up on this issue is that I have seen a few reports that users are not able to locate the "Convert to regular blocks" button.

**Comment by NewYorkerLaura** [comment]
All I have to add it to **STRONGLY** concur with everything @paaljoachim said above. We went from something with an elegantly simply and effective UX in WordPress 5.6.2 to massive confusion in its current state.  I'm am constantly helping people figure reusable blocks out in a multitude of Facebook groups and Slack channels. 

Speaking of the current iteration, It's also darned inconvenient to name my block in the sidebar!! Everyone misses the fact that that is where it moved to. It should be right above where it was.

This block really went from fantastic in 5.6.2 to making it a hard sell and making it take up more of my time.  I went from clients feeling empowered to make their own blocks to them constantly contacting us because they have either inadvertently wiped something out or they don't know what's going on.
